atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/HeartbeatService.java

```diff
@@ -61,7 +59,7 @@
         this.heartbeatExecutorService = null;
     }
 
-    void startBeatingForLock(long lockId) {
+    public synchronized void startBeatingForLock(long lockId) {
```

Comment:
Seems like we should never have multiple threads accessing this class.  Only one thread should ever be able to progress past waitForSchemaMutationLock().  Am I missing something here?